### DISCUSSION TITLE
Open language channel by default

### DISCUSSION THREAD

**Discussion by tatitatai** [ppy/osu#26043] [Ideas]
Detect the player's location/flag and open fitting language channel automatically. This will prevent new users talking in their native language in english-only channels

**Comment by Gabixel** [comment]
Maybe show it as a "suggested" channel, kind of like Discord does in some cases when you hide some. But I don't know if that would seem invasive as a thing:
![image](https://github.com/ppy/osu/assets/43073074/4aa7bab2-0c24-48ac-8bfa-87afbcf68d8f)

Maybe just joining it automatically is enough.

**Comment by ghost** [comment]
> Detect the player's location/flag

I disagree with this and would instead make the detection be based on the language setting of the client.
